commands/python/command/cmd.py

```diff
@@ -24,13 +31,38 @@ def cmd_args(args):
 
 def cmd(app_name, command_args, namespace=None):
     k = Kubernetes(namespace=namespace)
-    pod_name = k.get_pod_name("%s-commands" % app_name)
+    pod_name = k.get_pod_name(f"{app_name}-commands")
     logging.info("Command output below...")
+
     try:
-        # See if the commands container uses python3
-        executable = k.kub_exec(pod_name, "%s-commands" % app_name, 'which', 'python3',
-                                return_output=True).decode(sys.stdout.encoding).strip()
+        # In commands-base:2.0.0, we moved the commands.py script to /usr/local/bin/aladdin_cmd_v2
+        # and made it executable. It's expected that the implementer will provide a she-bang line to
+        # indicate which interpreter to use (or none at all if they are using a compiled executable)
+        k.kub_exec(
+            pod_name,
+            f"{app_name}-commands",
+            "/bin/bash",
+            "-c",
+            "test -x /usr/local/bin/aladdin_cmd_v2",
```

Comment:
I updated this to `/usr/local/bin/aladdin_cmd_v2` to future-proof us for potential future version changes. but maybe we leave it as `/usr/local/bin/aladdin_cmd` and deal with any future version changes by adding a version suffix in the future. I don't know how much more generic we can make this, so i don't anticipate many interface updates in the future. Maybe we start injecting arguments from aladdin as we call the command container?